### ISSUE TITLE
adding the plugin executes 10 of the 13 tasks when doing a gradle build

### ISSUE THREAD

**Issue by tessus** [crotwell/gradle-macappbundle#39]
I've added the following to my `build.gradle` file:

```
plugins {
    id "edu.sc.seis.macAppBundle" version "2.1.4"
}

macAppBundle {
    mainClassName = "..."
    icon          = "myApp.icns"
    volumeName    = "myApp"
    dmgName       = "myApp"
    javaProperties.put("apple.laf.useScreenMenuBar", "true")
    // Uncomment the next line to bundle the JRE with the Mac OS X application
    //bundleJRE     = true
}
```

When I run a `gradlew build`, all tasks from your bundle are run, except `codeSign`, `createAppZip`, `runSetFile`.

I never specified it anywhere that any of the tasks are run. The problem is that on Windows a `gradlew build` tries to run createDmg and fails (naturally).

Why are any MacAppBundle tasks run when doing a `gradlew build`?

Is there a way that none of the MacAppBundle tasks are run for a `gradlew build`?

**Comment by crotwell** [comment]
The tasks are run because "assemble" depends on them, basically if you add the macAppBundle plugin,you are saying that a app is an output of the build and so it is generated. I believe this is the expected behavior, for example if you use the war plugin, the war file will be generated on gradle build. If you don't want everything generated, you can always just call gradle jar to only build the jar file.

But the windows issue should be fixed in 2.1.5. Now assemble depends on createDmg on mac, but on createAppZip on windows and linux.

**Comment by tessus** [comment]
Hmm, so build includes assemble and assemble includes the 10 tasks. Ok, I'll try 2.1.5. Thanks.
